1 Week Preparation Kit/lonely_integer.py

```diff
@@ -0,0 +1,34 @@
+#!/bin/python3
+
+import math
+import os
+import random
+import re
+import sys
+
+#
+# Complete the 'lonelyinteger' function below.
+#
+# The function is expected to return an INTEGER.
+# The function accepts INTEGER_ARRAY a as parameter.
+#
+
+def lonelyinteger(a):
+    dicc = dict.fromkeys(a)
+    for value in dicc:
+        occurrences = a.count(value)
+        dicc.update({value: occurrences})
+    return list(dicc.keys())[list(dicc.values()).index(1)]
```

Comment:
Please save into variables some operations in order to make it more legible, for example `list(dicc.values()).index(1)` could be saved into a `lonely_int_index` variable

Reply:
Done